packages/react-components/src/templates/SharedResearchOutput.tsx

```diff
@@ -20,200 +22,71 @@ const cardsStyles = css({
   rowGap: `${36 / perRem}em`,
 });
 
-const additionalInformationListStyles = css({
-  listStyle: 'none',
-  margin: 0,
-  padding: `${6 / perRem}em 0`,
-});
-const additionalInformationEntryStyles = css({
-  display: 'flex',
-  justifyContent: 'space-between',
-  padding: `${6 / perRem}em 0`,
-  [`@media (max-width: ${mobileScreen.max}px)`]: {
-    flexDirection: 'column',
-  },
-});
-const additionalInformationValueStyles = css({
-  [`@media (max-width: ${mobileScreen.max}px)`]: {
-    marginTop: `${12 / perRem}em`,
-  },
-});
-const externalLinkStyle = css({
-  display: 'flex',
-});
-
 type SharedResearchOutputProps = Pick<
   ResearchOutputResponse,
-  | 'description'
-  | 'tags'
-  | 'accessInstructions'
-  | 'sharingStatus'
-  | 'asapFunded'
-  | 'usedInPublication'
-  | 'contactEmails'
-  | 'doi'
-  | 'rrid'
-  | 'accession'
-  | 'labCatalogNumber'
+  'description' | 'tags' | 'accessInstructions' | 'contactEmails'
 > &
   ComponentProps<typeof SharedResearchOutputHeaderCard> & {
     backHref: string;
-  };
+  } & ComponentProps<typeof SharedResearchAdditionalInformationCard>;
 
 const SharedResearchOutput: React.FC<SharedResearchOutputProps> = ({
   description,
   backHref,
   tags,
   accessInstructions,
-  sharingStatus,
-  asapFunded,
-  usedInPublication,
   contactEmails,
-  doi,
-  rrid,
-  accession,
-  labCatalogNumber,
   ...props
-}) => (
-  <div css={containerStyles}>
-    <BackLink href={backHref} />
-    <div css={cardsStyles}>
-      <SharedResearchOutputHeaderCard {...props} />
-      {(description || !!tags.length) && (
-        <Card>
-          {description && (
+}) => {
+  const isGrantDocument = (
+    ['Grant Document', 'Proposal'] as ResearchOutputType[]
+  ).includes(props.type);
+  return (
+    <div css={containerStyles}>
+      <BackLink href={backHref} />
+      <div css={cardsStyles}>
+        <SharedResearchOutputHeaderCard {...props} />
+        {((description && !isGrantDocument) || !!tags.length) && (
```

Comment:
I would move the `(description && !isGrantDocument)` to a `showDescription` and the `tags.length` to `showTags` and to facilitate understanding the logic happening here

Reply:
or `showNormalDescription` or default to differentiate for the Grant document's one

Reply:
there is also no need for `!!`

Reply:
I have removed the additional !!. The problem with trying to simplify the variable names is that there are three states. Grant Document | Other and then we show hide depending if description is there.